src/game/Game.java

```diff
@@ -71,6 +75,42 @@ public Game(Game g) {
 		this.moveTimer = g.getMoveTimer();
 	}
 
+	public void assignAi(boolean p1, boolean p2) {
```

Comment:
Why not just have an "assignp1AI" method and a "assignP2AI" method so that we dont have this primitive obsession code smell